### DISCUSSION TITLE
How can setup my database region as close as possible to where your application is being deployed

### DISCUSSION THREAD

**Discussion by hussamkhatib** [vercel/community#1428] [Help] [answered]
### Summary

I am using vercel for deploying our application and postgres database via [railway](https://railway.app/)  
I am concerned about the query speed, as it's insanely slow.  
I am guessing it is probably because our database region is far from where the application is served from. 

Possible solution I could think of:
There's a vercel integration for cockroachDB which can serve us postgres database. I am not sure if the integration alone solves the problem or the integration is just for making the setup easier.

**Comment by amyegan** [comment, marked as answer]
I don't have any advice specific to Postgres and Railway, but you can change your [default Serverless Functions region](https://vercel.com/docs/concepts/functions/serverless-functions/regions#) from your Project Settings. Region can also be set in a `vercel.json` config with the [`regions`](https://vercel.com/docs/concepts/projects/project-configuration#regions) property or by using the [`--regions`](https://vercel.com/docs/cli/deploy#regions) option on a CLI deployment. 

The guide on [using Databases with Vercel](https://vercel.com/guides/using-databases-with-vercel?query=database) describes a few strategies to help improve performance
